addons/mail/static/src/components/chat_window/chat_window.scss

```diff
@@ -23,44 +18,20 @@
         }
     }
 
-    &.o-mobile {
-        position: fixed;
-    }
-
-    &.o-fullscreen {
-        height: 100%;
-        width: 100%;
-    }
 }
 
 .o_ChatWindow_channelInvitationForm {
     min-height: 450px; // allow flex shrink smaller than content (but not too small)
 }
 
-.o_ChatWindow_header {
-    flex: 0 0 auto;
-}
-
-.o_ChatWindow_newMessageForm {
-    padding: map-get($spacers, 1);
-    margin-top: map-get($spacers, 1);
-    display: flex;
-    align-items: center;
-}
-
-.o_ChatWindow_newMessageFormInput {
-    flex: 1 1 auto;
-}
-
-.o_ChatWindow_newMessageFormLabel {
-    margin-right: map-get($spacers, 2);
-    flex: 0 0 auto;
-}
-
+.o_ChatWindow_header, 
+.o_ChatWindow_newMessageFormInput,
+.o_ChatWindow_newMessageFormLabel,
 .o_ChatWindow_thread {
-    flex: 1 1 auto;
+    flex-basis: auto;
 }
 
+
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
```

Comment:
https://github.com/odoo/odoo/pull/85839/files#diff-e5ea01996c5bd038041497470854a9c1cf33913cce0b4b536cb783f6696c160aL41

Can't we use `border-lg-top` ? I know that the resulting value will not be exactly 6px but, actually, keeping these values consistent across the ui would be good.

---

https://github.com/odoo/odoo/pull/85839/files#diff-e5ea01996c5bd038041497470854a9c1cf33913cce0b4b536cb783f6696c160aR47

You can conbine the selectors, so 

```scss
&.o-focused:not(.o-mobile)
```

---

https://github.com/odoo/odoo/pull/85839/files#diff-e5ea01996c5bd038041497470854a9c1cf33913cce0b4b536cb783f6696c160aR53

`border-0` ?

Reply:
For `.border-0`; `.o_Composer` is a different component altogether, child of `o_ThreadView/o_ChatWindow_thread`